conda_forge_tick/auto_tick.xsh

```diff
@@ -411,6 +411,52 @@ def add_rebuild_blas(migrators, gx):
                 cycles=cycles, obj_version=0))
 
 
+def add_replacement_migrator(migrators, gx, old_pkg, new_pkg, rationale):
+    """Adds a migrator to replace one package with another.
+
+    Parameters
+    ----------
+    migrators : list of Migrator
+        The list of migrators to run.
+    gx : graph
+        The conda-forge dependency graph.
+    old_pkg : str
+        The package to be replaced.
+    new_pkg : str
+        The package to replace the `old_pkg`.
+    rationale : str
+        The reason the for the migration. Should be a full statement.
+
+    """
+    total_graph = copy.deepcopy(gx)
+
+    for node, node_attrs in gx.nodes.items():
+        attrs = node_attrs['payload']
+        meta_yaml = attrs.get("meta_yaml", {}) or {}
+        bh = get_requirements(meta_yaml)
+        pkgs = set([old_pkg])
+        old_pkg_c = len(pkgs.intersection(bh)) > 0
```

Comment:
couldn't this just be `old_pkg in pkgs`? The current syntax would be useful if this supports multi-package replacement, but the code doesn't seem to support that.